### PR TITLE
Release/0.1.0

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

**Review by sameepsi — APPROVED** [review]

File: src/components/MoonpayModal/MoonpayModal.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <CustomModal
+      open={open}
+      onClose={onClose}
+      background='#fff'
+      overflow='hidden'
+    >
+      <div style={{ height: '100%', width: '100%', overflowY: 'auto' }}>
+        <iframe
+          title='moonpay'
+          allow='accelerometer; autoplay; camera; gyroscope; payment'
```
Comment (FriskyHamTitz) [inline src/components/MoonpayModal/MoonpayModal.tsx:20]: Why do we need to allow camera? Probably kyc? Add a comment please in a seperate pr

File: src/pages/LandingPage/HeroSection.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            })}
+          </Typography>
+        </Box>
+      ) : (
+        <Box my={1}>
+          <Skeleton variant='rect' width={400} height={72} />
+        </Box>
+      )}
+      <Typography variant='h5'>
+        Top Asset Exchange on the Polygon Network
```
Comment (FriskyHamTitz) [inline src/pages/LandingPage/HeroSection.tsx:68]: Should be in the messages section

File: src/pages/LandingPage/HeroSection.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            ethereum && !isSupportedNetwork(ethereum)
+              ? addMaticToMetamask()
+              : account
+              ? history.push('/swap')
+              : toggleWalletModal();
+          }}
+        >
+          {ethereum && !isSupportedNetwork(ethereum)
+            ? 'Switch to Polygon'
+            : account
```
Comment (FriskyHamTitz) [inline src/pages/LandingPage/HeroSection.tsx:90]: Should be on the messages

File: src/pages/LandingPage/SwapSection.tsx [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+            {tabIndex === SWAP_TAB ? (
+              <Swap currencyBg={palette.background.paper} />
+            ) : (
+              <AddLiquidity currencyBg={palette.background.paper} />
+            )}
+          </Grid>
+          <Grid item sm={12} md={6} className={classes.swapInfo}>
+            <Typography variant='h4'>
+              {tabIndex === SWAP_TAB
+                ? 'Swap tokens at near-zero gas fees'
```
Comment (FriskyHamTitz) [inline src/pages/LandingPage/SwapSection.tsx:156]: Put into locales

**Review by FriskyHamTitz — APPROVED** [review]

**Pull request merged**